saleor/plugins/webhook/plugin.py

```diff
@@ -3212,6 +3212,8 @@ def get_taxes_for_order(
                 self.requestor,
             )
 
+    # Note: this method is deprecated in Saleor 3.20 and will be removed in Saleor 3.21.
```

Comment:
Why did you add this warning for this handler but didn't add it to the rest of WebhookPlugin handlers? 🤔 This can be misleading as WebhookPlugin.get_shipping_methods_for_checkout( will be just moved to core, but it will work in the same way as previously.

Reply:
Yes, I think that's a mistake; I'll revert this. Deprecations are added in manager and base plugin.